app/controllers/exceptions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ExceptionsController < ActionController::Base
+
+  include LocaleSelector
+
+  skip_filter *_process_action_callbacks.map(&:filter)
+
+  before_filter :set_locale
+
+  def rescue_from
+    @exception = env["action_dispatch.exception"]
+
+    OpenStax::RescueFrom.perform_rescue @exception, self
+  end
```

Comment:
:) For some reason it was really hard to get it to render the exception page on a bad request (see the unknown route feature spec). I think even tutor-server doesn't handle those properly: https://tutor.openstax.org/%EB just gives a blank page.